tests/unit/components/buttons/__snapshots__/HdButton.spec.js.snap

```diff
@@ -51,7 +51,7 @@ exports[`HdButton should render component with btn--tertiary class 1`] = `
 
 exports[`HdButton should render the icon passed as prop 1`] = `
 <button class="btn btn--primary"><span class="btn__icon"><svg height="100%" width="100%" viewBox="222 126 53 53" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg">
-  <path class="color1" d="M272.48 137.62L258.61 150.5L244.75 141.58L254.65 126.73L272.48 137.62Z" fill="#1895FF"></path>
-  <path class="color2" d="M242.22 176.24L222.97 160.68L230.47 145.1L243.56 155.68L242.22 176.24Z" fill="#1C3553"></path>
+  <path xmlns="http://www.w3.org/2000/svg" class="color1" d="M272.48 137.62L258.61 150.5L244.75 141.58L254.65 126.73L272.48 137.62Z" fill="#1895FF"></path>
```

Comment:
Hey man, just out of curiosity, is there any specific reason to add `xmlns="http://www.w3.org/2000/svg"` to the path?

Reply:
Soo, this was just changed for the testing scenario, this is something related to this commit:

099b7f3 

Now we are using the preset coming from `vue-cli` for our tests, and behind the scenes, they are adding that to SVG parsing, but in the actual implementation, things are normal.